repo/db/albums.go

```diff
@@ -91,6 +91,13 @@ func (c *AlbumDB) DeleteAlbum(id string) error {
 	return err
 }
 
+func (c *AlbumDB) DeleteAlbumByName(name string) error {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+	_, err := c.db.Exec("delete from albums where name=?", name)
+	return err
+}
+
```

Comment:
sweet, i'd say add this method to the interface as well, but it's totally ripped up in my branch, so may as well skip it.